### DISCUSSION TITLE
migrations and seeders do not work if I run them from the same command in production mode

### DISCUSSION THREAD

**Discussion by alexv96** [spatie/laravel-multitenancy#407] [Q&A]
I have a command that creates tenants, and runs the command to make the migration and seeders to the new tenant, the issue is that it does not work if it is in a production environment only works if it is in the local environment. How could I make it work the same way without changing the environment every time I want to create tenants?

**Comment by denizgolbas** [comment]
It is exactly related about production mode. When you try seed command at your server you will be see a interaction like this:
```bash
**************************************
*     Application In Production!     *
**************************************

 Do you really wish to run this command? (yes/no) [no]:
```

Solution is putting and of the seed command this `--no-interaction` or `--force`
